src/Umbraco.Web.Common/Umbraco.Web.Common.csproj

```diff
@@ -26,8 +26,8 @@
       <PackageReference Include="NETStandard.Library" Version="2.0.3" />
       <PackageReference Include="Serilog.AspNetCore" Version="3.4.0" />
       <PackageReference Include="SixLabors.ImageSharp.Web" Version="1.0.0" />
-      <PackageReference Include="Smidge" Version="3.1.1" />
-      <PackageReference Include="Smidge.Nuglify" Version="2.0.0" />
+      <PackageReference Include="Smidge" Version="3.2.0-build.244" />
+      <PackageReference Include="Smidge.Nuglify" Version="3.2.0-build.244" />
```

Comment:
When do you expect these are released to the official nuget feed?. Do we need to upload them to our own alpha feed like we have done with examine, if we ship a new alpha?

Reply:
I can push this week